rust/src/sip/sip.rs

```diff
@@ -313,30 +445,79 @@ pub unsafe extern "C" fn rs_sip_probing_parser_tc(
     return ALPROTO_UNKNOWN;
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn rs_sip_probing_parser_tcp_tc(
+    _flow: *const Flow, _direction: u8, input: *const u8, input_len: u32, _rdir: *mut u8,
+) -> AppProto {
+    if input_len >= 3 && !input.is_null() {
```

Comment:
does it make sense to already send 3 bytes of data? It seems the parser will need quite a few more to be able to succeed

Reply:
agree, it doesn't make much sense. Removed it.